### DISCUSSION TITLE
history -d does not delete entries

### DISCUSSION THREAD

**Discussion by mimkorn** [ohmyzsh/ohmyzsh#10411] [Help] [answered]
Hi,

I first filed the question here https://apple.stackexchange.com/questions/430640/history-d-does-not-remove-history-entry-but-shows-time-of-history-entry but then found out that my terminal isn't using standard history, but omz_history

Using `history -d XXX` won't delete an entry, but instead shows time next to used commands.

Is this a bug? How can I delete an entry with omz_history?

Thanks.

**Comment by mcornella** [comment, marked as answer]
[Answered here](https://apple.stackexchange.com/a/430643/439389).
